cluster/test-deploy/gcp-crio/vars-origin.yaml

```diff
@@ -0,0 +1,97 @@
+# Package and target versions
+ 
+openshift_use_crio: true
+openshift_crio_enable_docker_gc: true
+
+deployment_type: origin
+ansible_pkg_mgr: yum
+inventory_ip_type: external
+
+openshift_disable_check: disk_availability,docker_image_availability,memory_availability,docker_storage
+openshift_additional_repos: [
+  {"name": "origin-pr", "baseurl": "{{ openshift_test_repo }}", "enabled": 1, "gpgcheck": 0},
+  {"name": "origin-pr-dependencies", "baseurl": "http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/", "enabled": 1, "gpgcheck": 0}
+  ]
+
+# URLs and certs
+
+openshift_master_cluster_public_hostname: api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+openshift_master_cluster_hostname: internal-api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+wildcard_zone: apps.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}
+openshift_master_public_api_url: "https://api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}/"
+openshift_master_public_console_url: "https://api.{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}.{{ public_hosted_zone }}/console"
+
+openshift_template_service_broker_namespaces: [ "openshift" ]
+# Disabled until bootstrapping is fixed
+ansible_service_broker_install: false
+openshift_enable_service_catalog: false
+template_service_broker_install: false
+
+console_port: 443
+internal_console_port: 8443
+openshift_master_api_port: "8443"
+openshift_master_console_port: "8443"
+openshift_node_port_range: 30000-32000
+openshift_node_open_ports: [{"service":"Master health check port","port":"8080/tcp"},{"service":"Router stats port","port":"1936/tcp"},{"service":"Allowed open host ports","port":"9000-10000/tcp"},{"service":"Allowed open host ports","port":"9000-10000/udp"}]
+
+# Authentication and authorization
+
+openshift_master_identity_providers: [{"name": "allow_all", "login": "true", "challenge": "true", "kind": "AllowAllPasswordIdentityProvider"}]
+
+# Post config setting sizes
+
+openshift_hosted_router_replicas: 1
+openshift_hosted_registry_replicas: 1
+openshift_schedulable: True
+
+openshift_hosted_registry_storage_gcs_keyfile: "{{ files_dir }}/gce.json"
+openshift_hosted_registry_storage_kind: object
+openshift_hosted_registry_storage_provider: gcs
+openshift_hosted_registry_storage_gcs_bucket: origin-ci-test-{{ lookup('env', 'INSTANCE_PREFIX') | mandatory }}
+
+container_runtime_docker_storage_type: overlay2
+
+provision_custom_repositories: [
+  {
+    "name": "dockertested",
+    "baseurl": "https://mirror.ops.rhcloud.com/enterprise/rhel/dockertested/x86_64/os/
```

Comment:
@smarterclayton who has access to add rpms to this repo?
cc: @lsm5

Reply:
@mrunalp so the thing with dockertested is it's not cri-o version specific. So having cri-o 1.9, 1.10, 1.11 in the same repo is sure to cause problems. We'll need additional version-specific yum repos specifically to handle cri-o and cri-tools, which is what the CentOS PaaS SIG provides already.

Reply:
and as for docker and other packages which aren't openshift version specific, I'm hoping we can use CentOS Virt SIG, so there would be no need to keep dockertested moving forward.

Reply:
I gotta get CentOS rel-eng to set them up.

Reply:
@mrunalp RE: https://cbs.centos.org/repos/paas7-openshift-origin310-candidate/x86_64/os/ there's no ssl or gpg checks to do, so a simple: sslverify: false and gpgcheck: 0 should suffice along with the repo link. Does that help?

Reply:
let me add cri-tools in there and get back..

Reply:
@lsm5 thanks, I'll try it out once you add cri-tools.